ansible/roles/common/templates/alias.bashrc

```diff
@@ -140,4 +140,4 @@ phelp() {
 }
 _phelp_help="Print this help"
 
-alias phttp="http --verify no --cert {{ unprivileged_homedir }}/.pulp/user-cert.pem"
+alias phttp="http"
```

Comment:
This didn't work before and has become useless but I figured I'd keep it around in case people were still using it.